modules/nf-core/igvreports/main.nf

```diff
@@ -8,7 +8,7 @@ process IGVREPORTS {
         'biocontainers/igv-reports:1.12.0--pyh7cba7a3_0' }"
 
     input:
-    tuple val(meta), path(sites)
+    tuple val(meta), path(sites), path(tracks), path(tracks_indicies)
     path genomeFasta //optional genome fasta file
```

Comment:
Can we not use camelCase please, use snake_case.
I think this should just be fasta though, to be consistent with everything else (will need changing elsewhere, and the command changing to e.g. fasta_command)
```suggestion
    path fasta 
```